Tidy up alerts by priority and make notifications for high priority.

Yann-pechard opened the pull request:
Its for manage groups alerts to have only one message which contains all alerts of a group.

Commit:
send all alerts for a group

Its for manage groups alerts to have only one message which contains all alerts of a group.

gregoiremahe commented on alertmanager_receiver.py:
```diff
@@ -18,10 +18,22 @@ def alert():
+    if priority in ["P4","P5"]:
+        notification = "true"
+
+    requests.get('https://api.telegram.org/bot'+TELEGRAM_TOKEN+'/sendMessage?chat_id='+TELEGRAM_CHAT_ID+'&text='+total_alerts+'&disable_notification=(notification)')
```
You should check status code before the function ends

gregoiremahe replied:
Better to use %s for building strings from variables

gregoiremahe replied:
Notification variable not loaded in url. Did you confused string with variable ?

gregoiremahe commented on alertmanager_receiver.py:
```diff
@@ -18,10 +18,22 @@ def alert():
-
-    requests.get('https://api.telegram.org/bot'+TELEGRAM_TOKEN+'/sendMessage?chat_id='+TELEGRAM_CHAT_ID+'&text='+body+'&disable_notification=true')
+    total_alerts = ""
+    priority= ""
```
Should be None instead of empty string

gregoiremahe commented on alertmanager_receiver.py:
```diff
@@ -18,10 +18,22 @@ def alert():
     body = urllib.parse.quote(body)
-
-    requests.get('https://api.telegram.org/bot'+TELEGRAM_TOKEN+'/sendMessage?chat_id='+TELEGRAM_CHAT_ID+'&text='+body+'&disable_notification=true')
+    total_alerts = ""
```
More understandable to name the variable message_to_send or something similar

gregoiremahe commented on alertmanager_receiver.py:
```diff
@@ -18,10 +18,22 @@ def alert():
+    notification = "false"
+    
+    for P in ["P1","P2","P3","P4","P5"]:
+        for alert in body["alerts"]:
```
TypeError : Body is a string URL parsed, not dict

gregoiremahe commented on alertmanager_receiver.py:
```diff
@@ -18,10 +18,22 @@ def alert():
 
     body = request.data.decode('utf8')
-    body = json.loads(body)['alerts'][0]['annotations']['description']
     body = urllib.parse.quote(body)
```
Why to URL parse body if you intend to use it as dictionnary later ?

gregoiremahe reviewed (changes requested):
That code is not working, and will leads to loop rebooting

Commit:
Fix errors

Errors gets print in logs when status code is not equal to 200 and fix some errors in code
